docs/api/1.x/records.rst

```diff
@@ -265,6 +264,49 @@ Retrieving stored records
             ]
         }
 
+.. _records-head:
+
+Count of stored records
+=======================
```

Comment:
side note: In theory, this applies to every resources (buckets, collections, groups, etc.) but I'm fine if we only document it for records...

Reply:
I saw what you did, perfect!